sdk/src/connections/target/rgb_sensor.cpp

```diff
@@ -580,3 +579,157 @@ aditof::Status RgbSensor::getInternalBuffer(uint8_t **buffer,
 aditof::Status RgbSensor::enqueueInternalBuffer(struct v4l2_buffer &buf) {
     return enqueueInternalBufferPrivate(buf);
 }
+
+void RgbSensor::setRw(int value) { this->Rw = value; }
```

Comment:
having a notation with m_ prefix let's us know we're dealing with a member of this instance. So this isn't really necessary.